ca.json

```diff
@@ -1,1990 +1,1990 @@
 {
 	"setting": {
-		"options": "Options",
-		"plugin": "Plugin",
-		"builtin-plugins": "Core plugins",
-		"plugin-options": "Plugin options",
-		"folder-path-example-placeholder": "Example: folder 1/folder 2",
-		"file-path-example-placeholder": "Example: folder/note",
-		"msg-restart-required": "Changing this option requires a restart to take effect.",
-		"editor": {
+		"options": "Opcions",
+    	"plugin": "Connector",
+    	"builtin-plugins": "Connectors principals",
+    	"plugin-options": "Opcions del connector",
+    	"folder-path-example-placeholder": "Exemple: carpeta 1/carpeta 2",
+    	"file-path-example-placeholder": "Exemple: carpeta/nota",
+    	"msg-restart-required": "Canviar aquesta opció requereix un reinici perquè tingui efecte.",
+    	"editor": {
```

Comment:
Please fix the indentation here.

Reply:
Corrected. I have done the commit again, is that correct? Sorry if I didn't have to do it.